lib/generators/qa/local/tables/tables_generator.rb

```diff
@@ -6,7 +6,11 @@ class TablesGenerator < Rails::Generators::Base
 
     def migrations
       generate "model qa/local_authority name:string:uniq"
-      generate "model qa/local_authority_entry local_authority:references label:string uri:string:uniq"
+      generate "model qa/local_authority_entry qa_local_authority:references label:string uri:string:uniq"
```

Comment:
How about just gsub in the generated migration? Then we wouldn't need to change the model.